Controller/src/main/java/org/apidb/irods/ProcessIrodsEvents.java

```diff
@@ -87,39 +92,47 @@ public class ProcessIrodsEvents {
 
   public static void main(String[] args) throws Exception {
 
-	  // The id of the project for which these events are intended.
+    String rawProjectIds = System.getenv(ENV_PROJECT_IDS);
     String projectId = System.getenv(ENV_PROJECT_ID);
+    List<String> projectIds;
+
+    // The id of the project for which these events are intended.
+    if (rawProjectIds != null) {
+      projectIds = Arrays.asList(rawProjectIds.split(","));
```

Comment:
do we need to protect against bad input?   what about a possible space char after the comma delimiter?